shared/pages/import/oauth.html

```diff
@@ -14,7 +14,7 @@
     <script type="text/javascript" src="/shared/pages/import/js/flow.js"></script>
     <script type="text/javascript" src="/shared/pages/import/js/oauth20.js"></script>
     <script type="text/javascript" src="js/parameters.js"></script>
-    <script type="text/javascript" src="/shared/pages/import/js/curtain.js"></script>
+    <script type="text/javascript" src="/shared/pages/import/js/curtain_panel.js"></script>
     <script type="text/javascript" src="/shared/pages/import/js/oauth_frame.js"></script>
```

Comment:
+1. Added in [1]. Thanks!

[1] https://github.com/mozilla-b2g/gaia/pull/30918/files#diff-04ca5032b1aa919588ac7e771e6dc0f1R13